pyews/pyews.py

```diff
@@ -60,7 +60,7 @@ def __init__ (self, user, pwd):
         self.pwd  = pwd
 
 class ExchangeService(object):
-    def __init__ (self):
+    def __init__(self):
         self.ews_ad = None
```

Comment:
I consistently follow a convention of having one space between method name and parameter list in the function definition, and none during invocation. This is a long standing practice I bring from early in my career, and I would like to maintain that convention in this code, please.